kpi/tests/test_mock_data_exports.py

```diff
@@ -247,28 +247,25 @@ class MockDataExportsBase(TestCase):
                 'schema': '1',
                 'survey': [
                     {
-                        'name': 'person',
+                        'type': 'begin_group',
+                        'name': 'people',
+                        'label': ['People'],
+                    },
+                    {
```

Comment:
What about add another form , e.g. 'Simple nested group' and add a test which uses this form explicitly?

Reply:
@noliveleger, this is the test I added in the previous PR that attempted to fix the filtering issue. So this is just extending that test a little further to cover the previous and new fix so I don't think it needs a new test form. I'm happy to add one if you think that's a better way forward though 👍

Reply:
That's fine